config/routes.rb

```diff
@@ -24,6 +24,7 @@
   post "/are-you-a-manufacturer" => "coronavirus_form/are_you_a_manufacturer#submit"
 
   get "/product-details" => "coronavirus_form/product_details#show"
+  get "/product-details/:id/delete" => "coronavirus_form/product_details#destroy"
```

Comment:
Can we make this a DELETE request?

Reply:
I would like that; as far as I can see the component only supports GETs.

https://components.publishing.service.gov.uk/component-guide/summary_list/with_edit_on_individual_items

Reply:
That's a shame

Reply:
One option would be to add an intermediary page, with "are you sure you want to delete this product?" with a form that makes a DELETE request. I'm not sure of the value of this for users, though.

Reply:
I'd just go for the MVP approach at the moment (with GET request)